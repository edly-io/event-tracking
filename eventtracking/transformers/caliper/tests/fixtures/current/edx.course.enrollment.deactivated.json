{
    "context": {
        "org_id": "edX",
        "host": "localhost:18000",
        "ip": "172.18.0.1",
        "client_id": null,
        "session": "44c2f6b53a6aaf210797a56391a443bc",
        "path": "/change_enrollment",
        "username": "edx",
        "agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36",
        "course_id": "course-v1:edX+DemoX+Demo_Course",
        "referer": "http://localhost:18000/dashboard",
        "user_id": 3,
        "accept_language": "en-US,en;q=0.9"
    },
    "name": "edx.course.enrollment.deactivated",
    "timestamp": "2020-07-15T06:39:23.898755+00:00",
    "data": {
        "course_id": "course-v1:edX+DemoX+Demo_Course",
        "user_id": 3,
        "mode": "audit"
    }
}
