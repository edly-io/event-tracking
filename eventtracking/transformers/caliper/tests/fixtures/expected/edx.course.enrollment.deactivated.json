{
    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
    "action": "Deactivated",
    "actor": {
        "id": "anonymous_id",
        "type": "Person"
    },
    "eventTime": "2020-07-15T06:39:23.898Z",
    "id": "urn:uuid:b52eb5c3-350f-45c2-9f03-aa642b7ad30d",
    "object": {
        "extensions": {
            "course_id": "course-v1:edX+DemoX+Demo_Course",
            "mode": "audit"
        },
        "id": "http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course",
        "type": "Membership"
    },
    "referrer": {
        "id": "http://localhost:18000/dashboard",
        "type": "WebPage"
    },
    "type": "Event"
}
